Bukkit/src/main/java/com/plotsquared/bukkit/listener/PlayerEvents.java

```diff
@@ -946,7 +946,7 @@ public void playerMove(PlayerMoveEvent event) {
             || area.isForcingPlotChat())) {
             return;
         }
-        if (plot.isDenied(plotPlayer.getUUID())) {
+        if (plot.isDenied(plotPlayer.getUUID()) && !plotPlayer.hasPermission("plots.admin.command.chat")) {
```

Comment:
The permission node should be a caption in `com.plotsquared.core.configuration.Captions`, then `com.plotsquared.core.util.Permissions.hasPermission(player, caption)` can be used.